server/src/main/java/io/druid/client/CachingClusteredClient.java

```diff
@@ -332,20 +329,8 @@ private void addSequencesFromServer(ArrayList<Pair<Interval, Sequence<T>>> listO
               final DruidServer server = entry.getKey();
               final List<SegmentDescriptor> descriptors = entry.getValue();
 
-              final QueryRunner clientQueryable = new MetricsEmittingQueryRunner(
-                  emitter,
-                  new Function<Query<T>, ServiceMetricEvent.Builder>()
-                  {
-                    @Override
-                    public ServiceMetricEvent.Builder apply(@Nullable final Query<T> input)
-                    {
-                      return toolChest.makeMetricBuilder(input);
-                    }
-                  },
-                  serverView.getQueryRunner(server),
-                  "query/node/time",
-                  ImmutableMap.of("server",server.getName())
-              );
+              // TODO: Make the HTTP emitter not have such locky behavior... then make this emit metrics again
+              final QueryRunner clientQueryable = serverView.getQueryRunner(server);
```

Comment:
This PR causes broker performance to be indistinguishable from 0.7.3. Without this patch we are seeing a 10x performance decrease in a high concurrency environment.